src/validator/client/validator.ts

```diff
@@ -0,0 +1,288 @@
+      /** @description successful operation */
+      200: {
+        content: {
+          readonly "application/json": components["schemas"]["VersionInfo"];
+        };
+      };
+    };
+  };
+  queryByStatement: {
+    /**
+     * Query the network
+     * @description Returns the results of a SQL read query against the Tabeland network
+     */
+    parameters: {
+      /**
+       * @description The SQL read query statement
+       * @example select * from healthbot_80001_1
+       */
+      /**
+       * @description The requested response format:
+       *  * `objects` - Returns the query results as a JSON array of JSON objects.
+       *  * `table` - Return the query results as a JSON object with columns and rows properties.
+       */
+      /** @description Whether to extract the JSON object from the single property of the surrounding JSON object. */
+      /** @description Whether to unwrap the returned JSON objects from their surrounding array. */
+      readonly query: {
+        statement: string;
+        format?: "objects" | "table";
+        extract?: boolean;
+        unwrap?: boolean;
+      };
+    };
+    responses: {
+      /** @description Successful operation */
+      200: {
+        content: {
+          readonly "application/json": Record<string, never>;
+        };
+      };
+      /** @description Invalid query/statement value */
+      400: never;
+      /** @description Row Not Found */
+      404: never;
+      /** @description Too Many Requests */
+      429: never;
+    };
+  };
+  receiptByTransactionHash: {
+    /**
+     * Get transaction status
+     * @description Returns the status of a given transaction receipt by hash
+     */
+    parameters: {
+      /**
+       * @description The parent chain to target
+       * @example 80001
+       */
+      /**
+       * @description The transaction hash to request
+       * @example 0x400508d7cc035b14cc53f64393a8dafcc55f66ad8f9b44d626744157337e2098
+       */
+      readonly path: {
+        chainId: number;
+        transactionHash: string;
+      };
+    };
+    responses: {
+      /** @description successful operation */
+      200: {
+        content: {
+          readonly "application/json": components["schemas"]["TransactionReceipt"];
+        };
+      };
+      /** @description Invalid chain identifier or transaction hash format */
+      400: never;
+      /** @description No transaction receipt found with the provided hash */
+      404: never;
+      /** @description Too Many Requests */
+      429: never;
+    };
+  };
+  getTableById: {
+    /**
+     * Get table information
+     * @description Returns information about a single table, including schema information
+     */
+    parameters: {
+      /**
+       * @description The parent chain to target
+       * @example 80001
+       */
+      /**
+       * @description Table identifier
+       * @example 1
+       */
+      readonly path: {
+        chainId: number;
+        tableId: string;
```

Comment:
Why is the `tableId` a string instead of a number?

Reply:
Because large integers... It should now be a string across the entire stack, save perhaps for the smart contract itself. But the cool thing there is that ethersjs has this nice "big int ish" type that supports strings and ints and bigints. Anyway, long story short, if you want consistent big ints, use strings.